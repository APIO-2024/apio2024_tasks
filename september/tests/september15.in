txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 1 1 2 4 5
6 5 4 2 1 3
6 5 4 2 3 1
5 6 4 2 1 3
6 5 4 2 1 3
5 6 4 2 1 3
10 5
0 1 1 3 4 4 5 6 8
9 7 2 8 5 6 4 3 1
9 7 2 8 5 6 3 4 1
7 9 8 2 5 6 3 4 1
9 7 8 2 5 6 4 3 1
7 9 2 8 5 6 4 3 1
7 5
0 0 1 3 4 5
2 5 4 6 1 3
5 4 2 1 6 3
5 4 2 6 1 3
5 2 4 6 1 3
5 4 6 2 3 1
6 5
0 0 2 3 4
1 5 4 3 2
1 5 3 4 2
5 1 4 3 2
5 1 3 4 2
1 5 3 4 2
