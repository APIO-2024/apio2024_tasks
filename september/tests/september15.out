7ckgnn4wyi495puj3ibqf81dqvapyv6b
4
6
1
3
